tests/unit/backend/wmg/api/test_v2.py

```diff
@@ -707,17 +714,19 @@ def test__filter_request_with_query_key_not_specified_in_api_spec_returns_400(se
 
     @patch("backend.wmg.api.v2.fetch_datasets_metadata")
     @patch("backend.wmg.api.v2.gene_term_label")
+    @patch("backend.wmg.api.v2.ethnicity_term_label")
     @patch("backend.wmg.api.v2.ontology_term_label")
     @patch("backend.wmg.api.v2.load_snapshot")
     def test__filter_request_with_filter_dims__returns_valid_filter_dims__base_case(
-        self, load_snapshot, ontology_term_label, gene_term_label, fetch_datasets_metadata
+        self, load_snapshot, ontology_term_label, ethnicity_term_label, gene_term_label, fetch_datasets_metadata
```

Comment:
@atarashansky - Should there be a test case for a request to `/filters` for comma separated `self_reported_ethnicity_ontology_term_id`?

```
filter: {
self_reported_ethnicity_ontology_term_ids: ["self_reported_ethnicity_ontology_term_id_1,self_reported_ethnicity_ontology_term_id_2"]
```

It is not clear what the `"self_reported_ethnicity_terms"` key in the response would contain because that involves a call to a rather dense function `find_dim_option_values()`: https://github.com/chanzuckerberg/single-cell-data-portal/blob/1a8f0992d80e97933072e2262e02712500cea382/backend/wmg/api/v2.py#L218C29-L218C29

Reply:
@atarashansky - Yea that is the conclusion I came to and [added this comment](https://github.com/chanzuckerberg/single-cell-data-portal/pull/6102#issuecomment-1783539492) stating that unit tests that test the conversion from `self_reported_ethnicity_ontology_term_id` to label _for multi-valued ethnicity ontology IDs_ are sufficient